package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "jiti",
-  "version": "1.6.4",
+  "version": "1.7.0",
```

This number is auto increased by build script 
```suggestion
  "version": "1.6.4",
```